src/main/webapp/components/Navbar.vue

```diff
@@ -89,8 +97,8 @@ module.exports = {
       const authRes = this.googleAuth.currentUser.get().getAuthResponse();
       const token = authRes.id_token;
       const expirationTime = authRes.expires_at;
-      fetch('/user-registration?idToken=' + token + '&exp=' + expirationTime, {
-        method: 'POST'
+      fetch('/user-signin?idToken=' + token + '&exp=' + expirationTime, {
+        method: "POST"
```

Comment:
We want to keep this as a call to '/user-registration'. There is no path for '/user-signin'.

Reply:
Ahh thanks for catching that. Definitely some files in this PR that shouldn't have been modified, hopefully I get them all with the clean up commit.